buildlib/azure-pipelines-pr.yml

```diff
@@ -13,6 +13,9 @@ resources:
       endpoint: ucfconsort_registry
     - container: fedora
       image: rdmz-harbor.rdmz.labs.mlnx/ucx/fedora33:1
+    - container: rhel76
+      image: rdmz-harbor.rdmz.labs.mlnx/swx-infra/x86_64/rhel7.6/builder:mofed-5.1-2.3.8.0
```

Comment:
can we also add following libs to docker, to have good coverage:
1. cuda 11.1 kit
2. xpmem-devel
3. fuse3-devel

Reply:
Done